apis/cloudstack/src/main/java/org/jclouds/cloudstack/domain/UsageRecord.java

```diff
@@ -330,19 +340,21 @@ protected ConcreteBuilder self() {
    private final boolean isSourceNAT;
    private final double rawUsageHours;
    private final String usage;
+   private final long size;
```

Comment:
Is this parameter returned *always*? I see it declared as `@Nullable`. If it can be missing, then declare the variable as a `Long` object (primitives can't be null and NPEs will appear); otherwise remove the nullable annotation.